lib/ChangeTypes.js

```diff
@@ -42,6 +42,12 @@ module.exports = {
   // e.g. x + 2 + x^2 + x + 4 -> x^2 + (x + x) + (4 + 2)
   COLLECT_LIKE_TERMS: 'COLLECT_LIKE_TERMS',
 
+  // MULTIPLYING/DIVIDING CONSTANT POWERS
+
+  // e.g. 10^2 * 10^3 -> 10^(2+3)
+  // e.g. 10^4 / 10^2 -> 10^(4-2)
+  COLLECT_CONSTANT_EXPONENTS: 'COLLECT_CONSTANT_EXPONENTS',
```

Comment:
just have the multiply one here because division is in the other PR - you can add the division one in the other PR